### PR TITLE
Hotfix/an0127 fix tests errors

### LINKED ISSUE #1: Fix tests that are broken
fix tests

### PR DESCRIPTION
**Pull request by ArthurNovaes**
closes #1

### CODE REVIEW

File: lib/validators/add_user_session_vote.rb [hunk 1]
```diff
@@ -6,16 +6,20 @@ class AddUserSessionVote
...
       @users = attrs[:users]
     end
 
     private
-    attr_accessor :closed, :users, :session_vote
+    attr_accessor :status, :users, :session_vote
 
     def session_closed
-      self.errors.add(:session_vote, "Can't add user on session closed") if @closed && @users
+      self.errors.add(:session_vote, "Can't add user on session closed") if @status == ::SessionVote.statuses[:finish]
```
Comment (valterbarros) [inline lib/validators/add_user_session_vote.rb]: I think you forget to use the method session_vote_finish here

**Review by valterbarros — COMMENTED** [review]
#20 Solve the simple comment and go close this issue brow

**Comment by valterbarros** [comment]
closes #20

File: lib/validators/add_user_session_vote.rb (hunk 1 above)
Comment (ArthurNovaes) [reply lib/validators/add_user_session_vote.rb]: done

**Comment by valterbarros** [comment]
go ahead merge this!!

**Pull request merged**